nasa-gibs/worldview
# WV-3137:  Add TEMPO layers

## Description

Fixes #WV-3137 .

Adds 13 TEMPO Level 2 and Level 3 layers


## How To Test

[Provide whatever information a reviewer might need to know in order to verify that the changes made in this PR are working as expected. If there are special build steps that need to be taken in order to get these changes to run (building while on a separate branch, running `npm ci`, etc) include them here.]

- For bugfixes: What steps need to be taken in the UI to verify the bug is fixed?
- For enhancements and features: What is the expected functionality being added/modified? How can a reviewer verify this?

1. Open with [these URL parameters](http://localhost:3000/?v=-318.30326193314073,-64.80956442363771,45.813402146583485,115.90018737889207&z=4&i=5&ics=true&ici=5&icd=6&l=Reference_Labels_15m(hidden),Reference_Features_15m(hidden),Coastlines_15m(hidden),TEMPO_L3_Ozone_Cloud_Fraction(hidden),TEMPO_L3_Ozone_Column_Amount(hidden),TEMPO_L2_Ozone_Column_Amount_Granule(hidden,count=1),TEMPO_L3_Cloud_Cloud_Pressure_Total(hidden),TEMPO_L2_Cloud_Cloud_Pressure_Total_Granule(hidden,count=1),TEMPO_L3_Cloud_Cloud_Fraction_Total(hidden),TEMPO_L2_Cloud_Cloud_Fraction_Total_Granule(hidden,count=1),TEMPO_L3_NO2_Vertical_Column_Troposphere(hidden),TEMPO_L3_NO2_Vertical_Column_Stratosphere(hidden),TEMPO_L2_NO2_Vertical_Column_Troposphere_Granule(count=1),TEMPO_L2_NO2_Vertical_Column_Stratosphere_Granule(count=1),TEMPO_L3_Formaldehyde_Vertical_Column(hidden),TEMPO_L2_Formaldehyde_Vertical_Column_Granule(hidden,count=1),Land_Water_Map(opacity=0.8),VIIRS_NOAA20_CorrectedReflectance_TrueColor(hidden),VIIRS_SNPP_CorrectedReflectance_TrueColor(hidden),MODIS_Aqua_CorrectedReflectance_TrueColor(hidden),MODIS_Terra_CorrectedReflectance_TrueColor(hidden)&lg=true&t=2024-05-21-T18%3A02%3A08Z) for Level 2 Nitrogen Dioxide layers
2. Check to make sure imagery displays, the descriptions are good, you should not be able to take an image snapshot as I have disabled them for now. 
3. Use [this link](http://localhost:3000/?v=-318.30326193314073,-64.80956442363771,45.813402146583485,115.90018737889207&z=4&i=5&ics=true&ici=5&icd=6&l=Reference_Labels_15m(hidden),Reference_Features_15m(hidden),Coastlines_15m(hidden),TEMPO_L3_Ozone_Cloud_Fraction(hidden),TEMPO_L3_Ozone_Column_Amount(hidden),TEMPO_L2_Ozone_Column_Amount_Granule(hidden,count=1),TEMPO_L3_Cloud_Cloud_Pressure_Total(hidden),TEMPO_L2_Cloud_Cloud_Pressure_Total_Granule(hidden,count=1),TEMPO_L3_Cloud_Cloud_Fraction_Total(hidden),TEMPO_L2_Cloud_Cloud_Fraction_Total_Granule(hidden,count=1),TEMPO_L3_NO2_Vertical_Column_Troposphere(hidden),TEMPO_L3_NO2_Vertical_Column_Stratosphere(hidden),TEMPO_L3_Formaldehyde_Vertical_Column(hidden),TEMPO_L2_Formaldehyde_Vertical_Column_Granule(hidden,count=1),Land_Water_Map(opacity=0.8),VIIRS_NOAA20_CorrectedReflectance_TrueColor(hidden),VIIRS_SNPP_CorrectedReflectance_TrueColor(hidden),MODIS_Aqua_CorrectedReflectance_TrueColor(hidden),MODIS_Terra_CorrectedReflectance_TrueColor(hidden)&lg=true&t=2024-05-22-T20%3A26%3A08Z) to check the other 11 layers
4. Check to make sure imagery displays, the descriptions are good, you should not be able to take an image snapshot as I have disabled them for now. 


## PR Submission Checklist

This is simply a reminder of what we are going to look for before merging your code.

- I have read the [CONTRIBUTING](https://github.com/nasa-gibs/worldview/blob/main/.github/CONTRIBUTING.md) doc
- I have added necessary documentation (if applicable)
- I have added tests that prove my fix is effective or that my feature works (if applicable)
- Any dependent changes have been merged and published in downstream modules (if applicable)

## Merging

Please use the `squash and merge` commit method unless each commit in your branch is vital to the commit history of main.

@nasa-gibs/worldview
